susemanager-cloud-setup.spec

```diff
@@ -0,0 +1,72 @@
+#
+# spec file for package susemanager-cloud-setup-storage
+#
+# Copyright (c) 2019 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+Name:           susemanager-cloud-setup
```

Comment:
s/susemanager-cloud-setup/susemanager-cloud-storage-setup/

We do not want to give the impression that this sets up SUMa in any way shape or form

Reply:
Ok. Should I also rename the repository?

Reply:
Package name changed. I guess the repository name isn't really that important but I can rename that too.

Reply:
Yes please